experiments/configs/shufflenetv2x100_classification_cifar10.yml

```diff
@@ -38,7 +38,7 @@ model: {
     freeze_backbone: False
   },
   preprocess_args: {
-    input_size: 32,
+    input_size: 224,
```

Comment:
you do know that cifar's original image size is 32px right?

Reply:
sorry i didn't aware that i modified this, will check it